### PR TITLE
travis_CI: add Travis CI to compile every commit

### PR DESCRIPTION
**Pull request by wudping**
move README to README.md, and add the coverity message into
it.

to implement [VIZ-9125](https://jira01.devtools.intel.com/browse/VIZ-9125) based Yizhou's patches.

Signed-off-by: wudping <dongpingx.wu@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,53 @@
...
+
 Apache License 2.0
 
 
 Overview
 --------
 libyami consists of several libraries:
+
     * 'codecparsers': it is bit stream parser,
+
```
Comment (xuguangxin) [inline README.md]: seems this is little illformatted. https://github.com/wudping/libyami/tree/TravicCI

**Review by xuguangxin — CHANGES_REQUESTED** [review]
please help change      https://github.com/01org/libyami-utils line too.

**Comment by wudping** [comment]
okay @xuguangxin .

**Comment by xuguangxin** [comment]
Seems we only build h264 encoder and some of decoders
See https://travis-ci.org/01org/libyami
Could you add more? Please also add v4l2 decoder and encoder

**Comment by wudping** [comment]
okay, @xuguangxin , Thanks.

**Review by xuguangxin — APPROVED** [review]

**Pull request merged**